draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -304,6 +340,13 @@ The REQUIRED "k" (key ID) parameter is a byte sequence that identifies which key
 the user agent wishes to use to authenticate. This can for example be used to
 point to an entry into a server-side database of known keys.
 
+## The a Parameter {#parameter-a}
+
+The REQUIRED "a" (public key) parameter is a byte sequence that contains the
+public key used by the server to validate the signature provided by the client.
+This avoids key confusion issues (see {{SEEMS-LEGIT}}). The encoding of the
+public key is described in {{context}}.
```

Comment:
Should we add text which describes what to do if the encoding of this parameter is invalid, or is that already covered elsewhere?

Reply:
I think that's covered by the `mismatch between key ID and provided public key` text (you don't need to check for invalid encoding, just `memcmp` with your known valid encoding), but this is making me want to make that normative. Will do in followup PR.

Reply:
Sounds good!